server/src/main/java/io/crate/expression/scalar/DateBinFunction.java

```diff
@@ -99,7 +99,8 @@ public Scalar<Long, Object> compile(List<Symbol> arguments, String currentUser,
     }
 
     @Override
-    public final Long evaluate(TransactionContext txnCtx, NodeContext nodeCtx, Input[] args) {
+    @SafeVarargs
+    public final Long evaluate(TransactionContext txnCtx, NodeContext nodeCtx, Input<Object> ... args) {
```

Comment:
Any benefit of this change over the `[]` (since we always have at list 3 elements) ?

Reply:
It avoids the unchecked warnings